packages/util/src/units.ts

```diff
@@ -1,6 +1,9 @@
 import { BIGINT_0, BIGINT_1 } from './constants.js'
+
 /** Easy conversion from Gwei to wei */
 export const GWEI_TO_WEI = BigInt(1000000000)
+export const WEI_TO_EITHER = BigInt(10 ** 18)
+export const WIE_TO_GWEI = BigInt(10 ** 9)
```

Comment:
This should be GWEI_TO_WEI, a constant which is declared above (although it is more clear to write that constant as `10 ** 9` to make it more readable). Also a typo: `WIE` -> `WEI`.